sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxSqaleXmlLoader.java

```diff
@@ -0,0 +1,186 @@
+/*

```

Comment:
Thx for the feedback. Problem with this SqaleXmlLoader is that most methods are private so you can't derive and extend functionality. Second point is that it is using InputStreamReader and not Reader. 

What do you mean with read to a temp file? How should this work?

> Am 07.06.2015 um 13:16 schrieb Jorge Costa <notifications@github.com>:
> 
> In sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxSqaleXmlLoader.java:
> 
> > @@ -0,0 +1,186 @@
> > +/*
> sucks a bit, in this case we need to have a integration test for this. or we might have the same issue as it happen with 0.9.4
> 
> —
> Reply to this email directly or view it on GitHub.
>